static/scss/answers/templates/vertical-grid.scss

```diff
@@ -167,9 +167,12 @@
     {
       .yxt-SpellCheck
       {
-        margin-top: calc(var(--yxt-base-spacing) / 2);
-        margin-bottom: var(--yxt-base-spacing);
+        margin-bottom: calc(var(--yxt-base-spacing) * 1.25);
         margin-left: var(--yxt-base-spacing);
+
+        @include bpgte(sm) {
```

Comment:
for my own edification, how are we deciding to use one of the the breakpoint mixins rather than the mobile breakpoint directly?

Reply:
I think in general we've been using the breakpoint mixins when possible, though I'm not sure if I agree with this. A lot of the time I find myself having to dife into the css plumbing to find out what breakpoint "sm" corresponds to. It's also pretty confusing that `bplte(xs)` corresponds to the same breakpoint as `bpgte(sm)` , that is 767 and 768px. Should I just use the breakpoints directly here?

Reply:
I imagine consulting had fancy reasons for breakpoint mixins, that we either didn't port over or don't take advantage of